.vscode/settings.json

```diff
@@ -1,3 +0,0 @@
-{
```

Comment:
I don't really know if we want this file here (I think @Shadoom7 snuck it in maybe .vscode should go in .gitignore) but let's not remove this in this PR.

Reply:
Ok I'll revert this.